code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/Identity.java

```diff
@@ -17,19 +17,26 @@
 import com.adobe.marketing.mobile.AdobeCallbackWithError;
 import com.adobe.marketing.mobile.AdobeError;
 import com.adobe.marketing.mobile.Event;
+import com.adobe.marketing.mobile.Extension;
 import com.adobe.marketing.mobile.ExtensionError;
 import com.adobe.marketing.mobile.ExtensionErrorCallback;
 import com.adobe.marketing.mobile.LoggingMode;
 import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.util.DataReader;
+import com.adobe.marketing.mobile.util.StringUtils;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
 /**
  * Defines the public APIs for the AEP Edge Identity extension.
  */
 public class Identity {
 
+	public static final Class<? extends Extension> EXTENSION = IdentityExtension.class;
+	private static final long CALLBACK_TIMEOUT_MILLIS = TimeUnit.SECONDS.toMillis(5);
```

Comment:
The documented API timeout is 500 ms. The timeout is short primarily because of getExperienceCloudId, which needs to be fast for customers implementing a hybrid application which contains a web view. As these APIs are time sensitive for customers, I'd suggest keeping the 500 ms value.
cc @emdobrin

Reply:
I did not realize that the documented value of timeout is 500ms and changed it to match the default max timeout for a response callback instead. 
I have now changed it to 500ms to match the documented contract 👍🏼